diff --git a/dist/index.edge.js b/dist/index.edge.js
index ecf717eaf52f944c5f71c7d2a331843aaaa1ad17..a47e0747c3f160951e99bc3af36880eb8024e727 100644
--- a/dist/index.edge.js
+++ b/dist/index.edge.js
@@ -18929,7 +18929,7 @@ function assertValue(v2, errorMessage) {
 // src/index.edge.ts
 var initializedResvg = initWasm(resvg_wasm);
 var initializedYoga = initYoga(yoga_wasm).then((yoga2) => Ll(yoga2));
-var fallbackFont = fetch(new URL("./noto-sans-v27-latin-regular.ttf", import.meta.url)).then((res) => res.arrayBuffer());
+var fallbackFont = fetch("https://cdn.jsdelivr.net/fontsource/fonts/geist-sans@latest/latin-500-normal.woff").then((res) => res.arrayBuffer());
 var ImageResponse = class extends Response {
   constructor(element, options = {}) {
     const result = new ReadableStream({
